devices/Vl6180X/VL6180X.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Diagnostics;
+using UnitsNet;
+
+namespace Iot.Device.VL6180X
+{
+    /// <summary>
+    /// Represents VL6180X.
+    /// </summary>
+    public sealed class VL6180X : IDisposable
```

Comment:
naming whise, we should follow our Pascal Case guidelines, so `Vl6180X`

Reply:
done